superset/cli.py

```diff
@@ -183,6 +180,9 @@ def load_examples_run(load_test_data):
     print('Loading [Multi Line]')
     data.load_multi_line()
 
+    print('Loading [Misc Charts] dashboard')
```

Comment:
`load_multi_line` also added slice into misc_dashboard. i switch the for these 2 load functions, so that misc_dashboards will have newly added slice.